packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -6,7 +6,11 @@
 import {Application, ApplicationConfig} from '@loopback/core';
 import {expect, createClientForHandler} from '@loopback/testlab';
 import {Route, RestBindings, RestServer, RestComponent} from '../..';
+import {IncomingMessage} from 'http';
+import {get as httpsGet, Agent} from 'https';
```

Comment:
I find it surprising to see `Agent` without any qualification (http vs. https). I am proposing the following change:

```ts
// here at the top
import * as https from 'https';

// usage
https.get(...)
new https.Agent(...)
```

Reply:
>  I find it surprising to see Agent without any qualification (http vs. https). I am proposing the following change:
> ```ts
> import * as https from 'https';
> ```

Seems like you forgot to make this change, PTAL again.